variables.tf

```
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+variable "project_id" {
+  description = "The project in which the resource belongs. If it is not provided, the provider project is used."
+}
+
+variable "job_name" {
```

This name is redundant.

```suggestion
variable "name" {
```

---

Fixed in https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/b42d51328a83653787d5973ab4ea414abd0a5bb1